pcsx2/USB/usb-pad/usb-mascon.cpp

```diff
@@ -0,0 +1,585 @@
+// SPDX-FileCopyrightText: 2002-2024 PCSX2 Dev Team
+// SPDX-License-Identifier: GPL-3.0+
+
+#include "usb-mascon.h"
```

Comment:
Please rename mascon (`usb-mascon.cpp / usb-mascon.h / DEVTYPE_MASCON / MasconDevice / MasconState`) to something suitable for both DDG and TS controllers like usb-train.cpp, DEVTYPE_TRAIN, etc